### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_Screenreader : NVDA is not announcing the install required dependencies wizard table data.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#11197]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#NVDA;#Benchmark;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: 2d989e1180f342d684a4e26f76b941843b7fc261
Date: 2020-07-03T05:33:38.245Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.3.1
Screenreader : NVDA 2019.3.1

**Repro Steps:**
1.Launch Azure Data Studio Insiders application.
2.Connect to server.
3.Right-click connection and click New Notebook4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows)
appears.
4.Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu.
5.Python Dependencies Wizard will appear.
6.Provide any Input file details & Next.
7.Start Screenreader as NVDA and observe whether NVDA is announcing the install required dependencies wizard table data or not.

**Actual:**
NVDA is not announcing the install required dependencies wizard table data.

**Expected:**
NVDA should announce the install required dependencies wizard table data.

**User Impact:**
Screen reader users will not get install required dependencies wizard table data.

**Attachment link for Reference:**
[11197_A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4869620/11197_A11y_AzureDataStudioStayHealthy_Notebook_Python.Dependencies.Wizard_Screenreader.zip)

**Comment by alanrenmsft** [comment]
@chlafreniere could you please take a look at this one?

**Comment by alanrenmsft** [comment]
I am not sure why nvda is not working, on Windows, we tried Narrator, and by pressing CAPS + Left/Right, I am able to navigate through all the elements, for the table, it will read row and column count as well as all the information inside it along with column and row index. general guideline we were told is we don't optimize for a particular screen reader if we are already adding all the accessibility attributes to the UI.
![image](https://user-images.githubusercontent.com/13777222/90173832-e01c4b80-dd59-11ea-9572-d5882f0f6940.png)

**Comment by VasuBhog** [comment]
Along with @alanrenmsft comment above. I was able to navigate to the table using VoiceOver on Mac as I was able to get through all elements of the table as well using CAPS + Right/Down Arrow keys to navigate within the table and out of the table. 

![image](https://user-images.githubusercontent.com/23587151/90175048-7c4f4e00-dd6c-11ea-86fb-fd8fdcded4fa.png)
![image](https://user-images.githubusercontent.com/23587151/90175850-b0773e80-dd6d-11ea-8ffe-e1fff5847a14.png)

**Comment by Rama8345** [comment]
Verified the issue at Azure Data Studio. the issue is not reproducible. So, closing the bug. Please find attachment for reference.
**Github Tags:**#Closed;
![11197_Closed](https://user-images.githubusercontent.com/115216656/221835431-2821c694-9110-4962-b55c-8a7ec4ebea8d.gif)
